debos.go

```diff
@@ -270,6 +271,12 @@ func main() {
 		panic(err)
 	}
 
+	// Initialise origins map
+	context.origins = make(map[string]string)
+	context.origins["artifactdir"] = context.artifactdir
+	context.origins["filesystem"] = context.rootdir
+	context.origins["recipedir"] = context.recipeDir
```

Comment:
Can we make this artifacts, and recipe, dropping the dir suffix as they're all directories :)

Reply:
done.